cookbooks/christensen_yuen_phase_function/christensen_yuen_phase_function.prm

```diff
@@ -0,0 +1,181 @@
+# This is a setup for convection in a 2D box with a phase transition
+# in the center, and corresponds to the setup of the Boussinesq cases
+# in Christensen & Yuen 1985.
+
+set Dimension                              = 2
+set End time                               = 2e8
+set Output directory                       = christensen_yuen
+
+# We use a nonlinear solver to make sure any nonlinearity that may be
+# associated with the phase transition is resolved, and we use a large
+# value for the Maximum nonlinear iterations to make it obvious if the
+# nonlinear solver does not converge. Usually, the nonlinear solver
+# should converge within 1-5 iterations.
+set Nonlinear solver scheme                = iterated Advection and Stokes
+set Max nonlinear iterations               = 500
+set Maximum time step                      = 1e5
+
+subsection Solver parameters
+  subsection Stokes solver parameters
+    set Stokes solver type = block GMG
+  end
```

Comment:
Finally some GMG models!👍

Reply:
Yes! I am looking forward to being able to run all my models with the GMG :smile: 
Traction boundary conditions will definitely help a lot.

Reply:
Traction boundary conditions should have worked a long time ago. Do you recall a specific problem?